docs/usage/building-a-bundle.md

```diff
@@ -18,7 +19,7 @@ Once you have your Runnable Bundle, you can run Atmo:
 > subo dev
 ```
 
-Atmo will start up serving on port 8080, and you will begin to see its structured logs in your terminal. Make a request to `POST localhost:8080/hello` with a request body to see it in action.
+Atmo will start up serving on port 8080, and you will begin to see its structured logs in your terminal. **Note** Check Docker to ensure only Atmo is running on the port or else the Atmo developement server will not start correctly. Make a request to `POST localhost:8080/hello` with a request body to see it in action.
```

Comment:
Should we give more generic advice for port conflicts here? Lots of different webdev things bind to :8080 by default and could cause trouble.

Either way, `docker ps` or the more generic `lsof -Pi | grep :8080` are quick and easy ways to check on UNIX-y platforms.

Reply:
ooo good point. Yeah I think that;s a great idea. Maybe saying that they should check for port conflicts if they see port error?